apis/v1alpha1/httproute_types.go

```diff
@@ -64,11 +64,12 @@ type HTTPRouteSpec struct {
 	//
 	// Hostname can be "precise" which is a domain name without the terminating
 	// dot of a network host (e.g. "foo.example.com") or "wildcard", which is
-	// a domain name prefixed with a single wildcard label (e.g. "*.example.com").
-	// The wildcard character '*' must appear by itself as the first DNS
+	// a domain name prefixed with a single wildcard label (e.g. "&ast;.example.com").
+	// The wildcard character '&ast;' must appear by itself as the first DNS
```

Comment:
Just trying to capture all changes from "*" in case we miss any.

Reply:
What happens if we do `*` (code escape in markdown)?

Reply:
Hard to tell from the diff but that paragraph is not removed; it's now generic to `AddressType` because any kind of address could be unsupported (at least in an API types sense).